resources/assets/js/components/JobBuilder/Details/JobDetails.tsx

```diff
@@ -474,47 +471,39 @@ const JobDetails: React.FunctionComponent<JobDetailsProps &
                 actions.setSubmitting(false); // Required by Formik to finish the submission cycle
               });
           }}
-          render={({
-            errors,
-            touched,
-            isSubmitting,
-            values,
-          }): React.ReactElement => (
+        >
+          {({ errors, touched, isSubmitting, values }): React.ReactElement => (
             <section>
-              <Form
-                id="job-information"
-                data-c-container="form"
```

Comment:
It doesn't need `data-c-container="form"` anymore?

Reply:
The container had extra padding on the right, which was causing it to look off-center. One of its parent elements already has `data-c-container="form"`.